Dockerfile-linux

```diff
@@ -0,0 +1,20 @@
+
+# BASE IMAGE
+FROM node:16
+
+# just get the package*.json files and run npm install. 
+# This is helpful for using build cache in subseqent build runs if caching is used when building 
+COPY package*.json ./
+RUN npm install
+
+# copy all app code
+COPY . ./
+
+# build typescript to javascript
+RUN npm run build
+
+#global installation of mizu
+RUN npm install -g
+
+# check if mizu is installed and working fine
+RUN mizu --version
```

Comment:
This doesn't need to be in the Dockerfile. But we do want to set mizu as the default entrypoint (with the CMD action.)

Reply:
Ha. yes not needed.
But just there to test mizu was successfully installed in that docker image.
if "RUN mizu --version" fails then docker build fails which is desirable.

Reply:
I will push changes for mizu as the default entrypoint.

Reply:
"if "RUN mizu --version" fails then docker build fails which is desirable."

Ah, good point. I didn't consider that.

Reply:
added default command
CMD ["mizu", "--help"]

Reply:
If the container is supposed to be used in the below steps of workflow for some tests then it wont make sense to add CMD ["mizu", "--help"] .
But this command make sense when there is some real human using the built container. (But we are not saving docker image here)